### DISCUSSION TITLE
Allow Cards or Notes on Board Instead of Only Draft Issues

### DISCUSSION THREAD

**Discussion by destiny-harris** [community/community#10149] [Projects and Issues]
Right now, you can only add a draft issue to make a note, unless I'm viewing incorrectly. It would be nice if you can add note cards to the boards as well.

**Comment by eddpascoal** [comment]
> Right now, you can only add a draft issue to make a note, unless I'm viewing incorrectly. It would be nice if you can add note cards to the boards as well.

Thanks for sharing this issue. I thought that I was doing something wrong as github [documentation](https://docs.github.com/en/github-ae@latest/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/adding-notes-to-a-project-board#adding-notes-to-a-project-board) is not clear on this.

**Comment by Katsute** [comment]
#4422
